repos/system_upgrade/el7toel8/libraries/rhsm.py

```diff
@@ -257,12 +296,12 @@ def get_existing_product_certificates(context, rhsm_info):
 
 
 @contextlib.contextmanager
-def switched_certificate(context, rhsm_info, cert_path, version):
+def switch_certificate(context, rhsm_info, cert_path):
```

Comment:
the rename is something what we should consider for deprecation process as well. But as told above, as we do not have set deprecation process yet, I am ok in this pr to go with *hard* remove. If you want, you can keep older name as well otherwise.

Reply:
Yeah, hard remove, similar to others in this PR.